File: Groovy/src/main/groovy/com/codevalue/archnext/BotConfig.java
package com.codevalue.archnext;

public class BotConfig {
    private static final String BOT_NAME_VAR_NAME = "BOT_NAME";
    private static final String BOT_TOKEN_VAR_NAME = "BOT_TOKEN";

    public static String getBotName() {
        String envValue = System.getenv(BOT_NAME_VAR_NAME);
        return envValue  == null ? "j25trivia_bot" : envValue;
    }

    public static String getBotToken() {
        return System.getenv(BOT_TOKEN_VAR_NAME);
    }
}

File: Kotlin/src/main/kotlin/com/codevalue/archnext/BotConfig.kt
package com.codevalue.archnext

object BotConfig {
    private const val BOT_NAME_VAR_NAME = "BOT_NAME"
    private const val BOT_TOKEN_VAR_NAME = "BOT_TOKEN"

    val botToken: String
        get() = System.getenv(BOT_TOKEN_VAR_NAME)

    val botName: String
        get() = System.getenv(BOT_NAME_VAR_NAME) ?: "j25trivia_bot"
}